### PR TITLE
 Changed json & php file save method 

### PR DESCRIPTION
**Pull request by adudekyeti**
Changed json & php file save method

### CODE REVIEW

File: app/Utils.php [hunk 1]
```diff
@@ -97,4 +97,34 @@ public static function isHtml(string $content): bool
...
+	 * @param string       $comment
+	 * @param int          $flag
+	 * @param bool         $return
+	 *
+	 * @return bool|int $value
+	 */
+	public static function saveToFile(string $pathDirectory, $content, string $comment = '', int $flag = 0, bool $return = false)
+	{
+		if (\is_array($content)) {
+			$content = \App\Json::encode($content);
```
Comment (mariuszkrzaczkowski) [inline app/Utils.php]:
```suggestion
			$content = \App\Utils::varExport($content);
```

File: app/PrivilegeFile.php [hunk 2]
```diff
@@ -31,7 +31,7 @@ public static function createUsersFile()
 			$users['id'][$row['id']] = array_map('\App\Purifier::encodeHtml', $row);
 			$users['userName'][$row['user_name']] = $row['id'];
 		}
-		file_put_contents(static::$usersFile, '<?php return ' . Utils::varExport($users) . ';');
+		Utils::saveToFile(static::$usersFile, Utils::varExport($users), '', 0, true);
```
Comment (mariuszkrzaczkowski) [inline app/PrivilegeFile.php]:
```suggestion
		Utils::saveToFile(static::$usersFile, $users, '', 0, true);
```

File: app/Utils.php (hunk 1 above)
Comment (mariuszkrzaczkowski) [reply app/Utils.php]: \yii\helpers\VarDumper::export

File: modules/OSSMail/models/AddressBook.php [hunk 3]
```diff
@@ -84,7 +84,7 @@ public static function getLastRecord()
 	 */
 	public static function saveLastRecord($record, $module)
 	{
-		file_put_contents(self::LAST_RECORD_CACHE, "<?php return ['module' => '$module','record' => $record];");
+		\App\Utils::saveToFile(self::LAST_RECORD_CACHE, "['module' => '$module','record' => $record]", '', 0, true);
```
Comment (mariuszkrzaczkowski) [inline modules/OSSMail/models/AddressBook.php]:
```suggestion
		\App\Utils::saveToFile(self::LAST_RECORD_CACHE, ['module' => $module,'record' => $record], '', 0, true);
```

File: app/Language.php [hunk 4]
```diff
@@ -705,7 +705,7 @@ public static function translationModify(string $language, string $fileName, str
 		if ($remove) {
 			unset($translations[$type][$label]);
 		}
-		if (false === file_put_contents($fileDirectory, Json::encode($translations, JSON_PRETTY_PRINT))) {
+		if (false === \App\Json::save($fileDirectory, $translations)) {
```
Comment (rskrzypczak) [inline app/Language.php]:
```suggestion
		if (false === Json::save($fileDirectory, $translations)) {
```

**Pull request merged**